code-of-conduct.md

```diff
@@ -30,6 +30,12 @@ The OpenContainers team reserves the right to deny participation any individual
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct.
 By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing this project.
 
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the code of conduct team at codeofconduct@opencontainers.org. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The code of conduct team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
This seems like a partial code of conduct -- have seen other code of conducts which include this language as well.. "in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, making comments, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.."